src/core/response.ts

```diff
@@ -0,0 +1,43 @@
+import { MessageReply } from './message-reply';
+
+type ResponseStatus = {
+  code: number
+  message: string
+};
+
+type ResponseOpts = {
```

Comment:
For consistency with the other comments taken.
```suggestion
type ResponseOptions = {
```

Reply:
https://github.com/TBD54566975/dwn-sdk-js/commit/79558b120f081869b43b92826b673d788cec72df